teams/project/builds-tools.yml

```diff
@@ -0,0 +1,36 @@
+# yaml-language-server: $schema=../sub-teams-schema.json
+name: builds-tools
+description: Team to work on conda-build and other build tools in the conda ecosystem
+charter: project
```

Comment:
I think a better approach would be to have something like 

```
type: sub-team OR project
```

Then `charter` should only be available for sub-teams